plugins/modules/vmware_dvswitch.py

```diff
@@ -653,10 +653,8 @@ def update_dvswitch(self):
                 changed_version = True
                 spec_product = self.create_product_spec(self.switch_version)
         else:
-            results['version'] = self.vcenter_switch_version
```

Comment:
We don't need to define `self.vcenter_switch_version` anymore in the constructor.

Reply:
Yes, you're right. Sorry, I didn't realize this isn't used anywhere else.